lib/bitcoind-zeromq.js

```diff
@@ -90,3 +95,21 @@ BitcoindZeroMQ.prototype.close = function() {
 		this.sock = null;
 	}
 };
+
+BitcoindZeroMQ.prototype.initializeStatusCheck = function() {
+
+	var initializeStatusCheck = _.bind(this.initializeStatusCheck, this);
+	var uri = 'https://' + this.options.statusUrl;
+
```

Comment:
Should use `setInterval` and store the interval reference on the BitcoindZeroMQ instance so that it can be cleared when `close` is called.